x-pack/plugins/observability/server/assets/component_templates/slo_summary_mappings_template.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SLO_RESOURCES_VERSION } from '../constants';
+
+export const getSLOSummaryMappingsTemplate = (name: string) => ({
+  name,
+  template: {
+    mappings: {
+      properties: {
+        slo: {
+          properties: {
+            id: {
+              type: 'keyword',
+              ignore_above: 256,
+            },
+            revision: {
+              type: 'long',
+            },
+            instanceId: {
+              type: 'keyword',
+              ignore_above: 256,
+            },
+          },
```

Comment:
💡 I think we should also add the following fields from the rollup data:
- budgetingMethod
- timeWindow.duration
- timewindow.type
So we can search and filter in the summary index by these fields as well.

What do you think?

Reply:
Added in https://github.com/elastic/kibana/pull/161392/commits/2268d612f162f82a224332ae8a76465f70d5e1cf